config/test.config.js

```diff
@@ -0,0 +1,9 @@
+const SECRETS = require('./secrets.json'); // nodejs will auto read json
+
+module.exports = {
+  NODE_ENV: process.env.NODE_ENV || 'development',
+  HOST: process.env.HOST || 'http://localhost',
+  PORT: process.env.PORT || 8081,
+  DB_URL: `${SECRETS.mongodb_dev.db_url}test`,
```

Comment:
We shouldn't concat 'test' to the string returned from `SECRETS.mongodb_dev.db_url`. Rather add a new object to secrets.json and respectively secrets-example.json (for others to implement their version of secrets.json correctly) - something like this:

`
    "mongodb_test": {
        "_comment": "Database URL path for test",
        "db_url": "mongodb://localhost/app_db_test"
    },
`

This allows us to manage the paths for prod, dev, and test independently and directly through secrets.json instead having to go into the config and make changes if we want to modify URI in the future. For instance I'm playing around with mlab and thus appending 'test' to dev url will fail since I have a specific URI I need to pass along to connect to the remote db.